config/packages/prod/monolog.yaml

```diff
@@ -1,9 +1,15 @@
 monolog:
     handlers:
         main:
+            type: fingers_crossed
+            action_level: error
+            handler: nested
+            excluded_http_codes: [404, 405]
+            buffer_size: 50 # How many messages should be saved? Prevent memory leaks
+        nested:
             type: stream
             path: "%kernel.logs_dir%/%kernel.environment%.log"
-            level: error
+            level: debug
```

Comment:
shouldn't this still be `error`?

Reply:
No because the `finger_crossed` logger just holds on to everything unless there is an error. If there is one we want the debug logs too. (he says having just spent an hour figuring this out)